docs/src/_guide/orchestration.md

```diff
@@ -13,34 +13,37 @@ When a new pipeline schedule is created using the [UI](/reference/ui) or [CLI](/
 
 ## Create a Schedule
 
-### Specifying extractors and loaders
+### Scheduling predefined jobs
 
-To regularly schedule your ELT to run, use the ["Pipelines" interface in the UI](/reference/ui#pipelines), or the [`meltano schedule` command](/reference/command-line-interface#schedule):
+To regularly schedule your pipeline to run first define it as a [job](/reference/command-line-interface#job) within your project.
+Then you can schedule it directly using the `meltano schedule add` command:
 
 ```bash
-meltano schedule add [SCHEDULE_NAME] --extractor [EXTRACTOR_NAME] --loader [TARGET_NAME] --interval [INTERVAL]
-```
-
-Example:
+# Define a job
+meltano job add tap-gitlab-to-target-postgres-with-dbt --tasks "tap-gitlab target-postgres dbt:run"
```

Comment:
does this need to be "dbt:run" or "dbt-prostgres:run" ? Mostly asking because I have this outstanding PR from my  Getting Started guide run through: https://github.com/meltano/meltano/pull/6092

Reply:
I think it would depend on which transformer you have installed in your project. I think you _can_ still use plain dbt instead of an adapter-specific version. But since in 2.0 we're also encouraging users to follow dbt best practices and use the adapter-specific version I think that yeah, changing this to `dbt-postgres` would be better, especially if that's what we're using in the getting started guide.